data/core/logview.lua

```diff
@@ -122,12 +123,33 @@ function LogView:on_mouse_pressed(button, px, py, clicks)
 end
 
 
+function LogView:on_mouse_wheel(y, x)
+  if 0 < self.scroll.to.y then
+    if not self.user_scroll then
+      self.user_scroll = { x = self.scroll.to.x, y = self.scroll.to.y }
+    else
+      self.user_scroll.y = self.scroll.to.y
+    end
+  else
+    self.user_scroll = nil
+  end
+end
```

Comment:
Do you need to check for scroll here? Couldn't you just check the current `self.scroll.to` in `LogView:update`?

Reply:
yes, works and is cleaner, thanks.